src/pow.cpp

```diff
@@ -109,13 +109,18 @@ unsigned int static DarkGravityWave(const CBlockIndex* pindexLast, const CBlockH
         if (pindexLast->nChainWork >= UintToArith256(uint256S("0x000000000000000000000000000000000000000000000000003ff00000000000"))
             // and immediately on devnet
             || !params.hashDevnetGenesisBlock.IsNull()) {
+            // recent block is more than 10 minutes old
             if (pblock->GetBlockTime() > pindexLast->GetBlockTime() + params.nPowTargetSpacing*4) {
                 arith_uint256 bnNew = arith_uint256().SetCompact(pindexLast->nBits) * 10;
                 if (bnNew > bnPowLimit) {
                     bnNew = bnPowLimit;
                 }
                 return bnNew.GetCompact();
             }
+            // recent block is more than 2 hours old
```

Comment:
Will it ever get here, or would the condition above catch it first?

> pblock->GetBlockTime() > pindexLast->GetBlockTime() + params.nPowTargetSpacing*4

(Should this block be moved up above?)

Reply:
@UdjinM6 nvm, I see you caught it too

Reply:
yep, 28be742 :)